packages/nvidia_nat_llama_index/src/nat/plugins/llama_index/embedder.py

```
@@ -28,30 +29,35 @@ async def azure_openai_llama_index(embedder_config: AzureOpenAIEmbedderModelConf
 
-    yield client
+        yield client
 
 
 @register_embedder_client(config_type=NIMEmbedderModelConfig, wrapper_type=LLMFrameworkEnum.LLAMA_INDEX)
 async def nim_llama_index(embedder_config: NIMEmbedderModelConfig, _builder: Builder):
 
     from llama_index.embeddings.nvidia import NVIDIAEmbedding  # pylint: disable=no-name-in-module
 
+    if not embedder_config.verify_ssl:
+        raise ValueError("verify_ssl is currently not supported for NVIDIAEmbedding.")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)